hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -76,6 +76,11 @@
   public static final String HOODIE_PROPERTIES_FILE = "hoodie.properties";
   public static final String HOODIE_PROPERTIES_FILE_BACKUP = "hoodie.properties.backup";
 
+  public static final ConfigProperty<String> DATABASE_NAME = ConfigProperty
+      .key("hoodie.database.name")
+      .noDefaultValue()
+      .withDocumentation("Database name that will be used for incremental query.");
```

Comment:
this is not just for incremental query now, right? spark sql also uses it. Better to add more info here to explain the use cases of this config, which will show up in the website for users to understand.

Reply:
Although spark sql will use it, it is only for incremental query at present.So do we need to change here?

Reply:
it will be helpful to explain the present use case more, e.g., when and why users should set it.

Reply:
I have revised the description, but it may be because my English level is not very good. If not, I hope you can provide a reference

Reply:
new version looks good.